src/test/e2e/25_helm_test.go

```diff
@@ -8,18 +8,37 @@ import (
 	"fmt"
 	"os/exec"
 	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
-var helmChartsPkg string
+func TestHelmReleaseHistory(t *testing.T) {
+	outputPath := t.TempDir()
+	localTgzChartPath := filepath.Join("src", "test", "packages", "25-helm-release-history")
+	_, _, err := e2e.Zarf(t, "package", "create", localTgzChartPath, "-o", outputPath, "--confirm")
+	require.NoError(t, err)
+
+	packagePath := filepath.Join(outputPath, fmt.Sprintf("zarf-package-helm-release-history-%s-0.0.1.tar.zst", e2e.Arch))
+	for range 10 {
```

Comment:
Can you make this 11 or 12, since we changed the max this test would still work on main